run_unix.yml

```diff
@@ -0,0 +1,417 @@
+name: Run Unix
```

Comment:
this file should be removed I think, looks like copypaste

Reply:
But this is the file I edited to add my build and CI commands. Why do you want me to remove it?

Reply:
Because it should be .github/workflows/run_unix.yaml, bit this one is a copypaste in root folder

Reply:
My bad. Removed.